File: 2.POO/Resto/Panaderia/Primera version/Java/Panaderia.java
import java.util.Scanner;
public class Panaderia {
    public static void main(String[] args) {
        Scanner mex= new Scanner(System.in);

        Panaderio P1 = new Panaderio();
        System.out.println("Ingresa el nombre del pan: ");
        P1.nombre= mex.nextLine();
        System.out.println("Ingresa un codigo para el pan: ");
        P1.codigo= mex.nextLine();
        System.out.println("Ingresa la cantidad del pan: ");
        P1.cantidad= mex.nextDouble();
        System.out.println("Ingresa el precio del pan: ");
        P1.costo= mex.nextDouble();

        System.out.println("Nombre del pan: " + P1.nombre);
        System.out.println("Codigo del pan: " + P1.codigo);
        System.out.println("Cantidad del pan: " + P1.cantidad);
        System.out.println("Precio del pan: $" + P1.costo);
        System.out.println("Ganancia del pan: $" + (P1.cantidad * P1.costo));
    } 
}

File: 2.POO/Resto/Panaderia/Primera version/C++/Panaderia.cpp
#include <iostream>
#include <cstdlib>
using namespace std;
#include "Panaderia.h"

int main() {
    Panaderia P1;
    
    cout << ("\n");
    system("color 0c");
    cout << "Ingresa el nombre del pan: ";
    getline(cin,P1.nombre);
    cout << "Ingresa un codigo para el pan: ";
    getline(cin,P1.codbarra);
    cout << "Ingresa la cantidad del pan: ";
    cin >> P1.cantidad;
    cout << "Ingresa el precio del pan: ";
    cin >> P1.costo;
    cout << "\n";

    cout << "Nombre del pan: " << P1.nombre << "\n";
    cout << "Codigo: " << P1.codbarra << "\n";
    cout << "Cantidad del pan: " << P1.cantidad << "\n";
    cout << "Precio: $" << P1.costo << "\n";
    cout << "Ganancias: $" << P1.costo * P1.cantidad << "\n";

    cout << "\n \n";
    cout << "------------------------------------------------------------------------------------------\n";
	cout << "Creador: Jesus A. Mex Damian 15/02/2023.";
    return 0;
}